src/render_tasks/d3d12_deferred_composition.hpp

```diff
@@ -26,11 +26,12 @@ namespace wr
 
 		bool is_hybrid;
 		bool is_path_tracer;
+		bool ao_enabled;
 	};
 
 	namespace internal
 	{
-
+		const unsigned int output_index = 7;
```

Comment:
Hardcoded value? We have a system to help with this I believe.

Reply:
Unused variable I accept all responsibilities but am clueless to why that's in there...

Reply:
Removed now